lib/include/event-driven/vDrawSkin.h

```diff
@@ -0,0 +1,1159 @@
+/*
+ *   Copyright (C) 2017 Event-driven Perception for Robotics
+ *   Author: arren.glover@iit.it
+ *           luca.gagliardi@iit.it
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation, either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef __VDRAWSKIN__
+#define __VDRAWSKIN__
+
+#include <string>
+#include <opencv2/opencv.hpp>
+#include <list>
+#include <yarp/os/all.h>
+#include "event-driven/vDraw.h"
+
+namespace ev {
+
+class PAD{
+protected :
+
+    const double DEG2RAD=M_PI/180.0;
+    int nTaxels;
+
+    double *dX,*dY;
+
+    inline std::list<unsigned int> vectorofIntEqualto(const std::vector<int> vec, const int val)
+    {
+        std::list<unsigned int> res;
+        for (size_t i = 0; i < vec.size(); i++)
+        {
+            if (vec[i]==val)
+            {
+                res.push_back(i);
+            }
+        }
+        return res;
+    }
+
+    std::map<int, std::vector<double> > data;
+
+public:
+    int xoffset,yoffset;
+    int scaling =4, radius = 6;
+    int noise = 2500;
+    int max_value = 15000;//good for default flat skin patch
```

Comment:
It should be sufficient to initialize the values using a list initialization.
```suggestion
    int scaling{4};
    int radius{6};
    int noise{2500};
    int max_value{15000};//good for default flat skin patch
```